tsconfig.json

```diff
@@ -20,7 +20,6 @@
     "skipLibCheck": true,
     "sourceMap": true,
     "strict": false,
-    "suppressImplicitAnyIndexErrors": true,
```

Comment:
but why did you turned off this rule? do you have some ts errors?

Reply:
![Screenshot_1](https://github.com/mate-academy/qa_cypress_e2e_settings_pom_local/assets/124871613/d913e9f8-0cc6-4d80-a646-5ab3f6133ac8)

Reply:
mentor advised to remove this rule, but if you have a solution to this problem I would love to hear from you!
![Screenshot_2](https://github.com/mate-academy/qa_cypress_e2e_settings_pom_local/assets/124871613/d5a48fe9-12a9-4f9f-8deb-86797c5c0c51)